### ISSUE TITLE
[Broken Website] okcupid.com

### ISSUE THREAD

**Issue by callaloo** [darkreader/darkreader#9716]
# Broken Website Report

## Website Address

okcupid.com
## Steps To Reproduce

Just create an account, even a dummy one, and once signed up and verified, try editing your profile.
## Problematic Observation

Form fields appear dark but the text is dark too.
## System Information

- OS: Windows 
- Browser: Brave (latest) 
- Dark Reader Version: 4.9.52

**Comment by Gusted** [comment]
I'm not able to reproduce this bug as this requires an account(thus quite a lot of PPI, some which cannot be faked). Adding help wanted label.

**Comment by callaloo** [comment]
> PPI

What's this in full? Anyways, this was reported to me by someone and I reproduced it using an account I later deleted.

**Comment by Gusted** [comment]
Email seems to be avoidable by certain low-profile temp mail providers. But phone numbers are not possible to provide a stub/public one.

**Comment by callaloo** [comment]
You are right. I tried the 10 minutes phone numbers online but none worked.
